### PR TITLE
Build using openjdk8

### PR DESCRIPTION
**Pull request by antoine777**
see [comment](https://github.com/spring-projects/spring-ldap/issues/489#issuecomment-606268597)

### CODE REVIEW

File: core/javacc.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@
 task downloadJavacc {
     ext.file = "javacc-5.0.tar.gz"
     ext.dest = file("$buildDir/javacc/$file")
-    ext.url = "https://javacc.org/downloads/$file"
+    ext.url = "http://download.openpkg.org/components/cache/javacc/$file"
```
Comment (rwinch) [inline core/javacc.gradle:5]: Is there a reason that this was updated for using openjdk8? If we do this we'd like this to be a URL that works over `https`.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=http\://services.gradle.org/distributions/gradle-3.1-all.zip
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
```
Comment (rwinch) [inline gradle/wrapper/gradle-wrapper.properties:6]: Good catch here. I'd prefer things to be distinct commits so I went ahead and pushed a separate commit to fix this.

File: core/javacc.gradle (hunk 1 above)
Comment (antoine777) [reply core/javacc.gradle:5]: No reason with openjdk8. 
It is another issue with the gradle build (last successful buld was 1 year ago). 
I didn't found an URL over `https` which serve `javacc-5.0.tar.gz` (i don"t know why this version does not exist on the new javacc download page https://github.com/javacc/javacc/releases)

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (antoine777) [reply gradle/wrapper/gradle-wrapper.properties:6]: OK

File: core/javacc.gradle (hunk 1 above)
Comment (rwinch) [reply core/javacc.gradle:5]: Let's split this up into another PR. The openjdk8 is a no brainer. It appears due to this not being available over https, javacc may take a bit more work.

**Comment by rwinch** [comment]
Thanks for the Pull Request! This is now merged into master via 50dae66a8c2de15f905048fc2043547d3772b223 :smile:

**Pull request closed without merging**